CODING_GUIDELINES.md

```diff
@@ -211,44 +217,6 @@ a += "b"
   + "d";
 ```
 
-* Initializers
-
-We allow brace enclosed initializers only for aggregates and arrays/containers.<br />
-Brace enclosed initializer MUST be used with equality sign if it follows the variable declaration.<br />
-Brace enclosed initializer MUST be additionally enclosed in parentheses if it is used in constructor initialization list.<br />
-Some valid use cases:
-```c++
-// aggregate
-Person john = { "John", "Smith", 21 };
-Person *john = new Person { "John", "Smith", 21 };
-
-// array
-int array[] = { 1, 2, 3, 4 };
-
-// container
-QHash<QString, QString> map = {
-    { "key1", "value1" },
-    { "key2", "value2" }
-);
-
-// member array
-SomeClass::SomeClass(BaseClass *parent)
-    : BaseClass(parent)
-    , m_someArrayMember({ 1, 2, 3, 4 })
-{
-}
-
-// return from function
-Person getPersonByName(const QString &name)
-{
-    // do something
-    return { name, surname, age };
-}
-
-// function argument
-doSomething({ name, surname, age }, someOtherArg);
-```
-
 * **auto** keyword
 
 We allow the use of the **auto** keyword only where it doesn't break the readability of the code (i.e. either we can gather enough information about the type from the right part of the expression, or we do not need to know the exact type), or where it is strictly necessary (for example, to compute the type of a lambda, etc.).<br />
```

Comment:
Thank you, Nyrox!

On 13 April 2016 at 01:47, LordNyriox <notifications@github.com> wrote:

> In CODING_GUIDELINES.md
> <https://github.com/qbittorrent/qBittorrent/pull/5083#discussion_r59473487>
> :
>
> > -    : BaseClass(parent)
> > -    , m_someArrayMember({ 1, 2, 3, 4 })
> > -{
> > -}
> > -
> > -// return from function
> > -Person getPersonByName(const QString &name)
> > -{
> > -    // do something
> > -    return { name, surname, age };
> > -}
> > -
> > -// function argument
> > -doSomething({ name, surname, age }, someOtherArg);
> > -```
> > -
> >  * **auto** keyword
> >
> >  We allow the use of the **auto** keyword only where it doesn't break the readability of the code (i.e. either we can gather enough information about the type from the right part of the expression, or we do not need to know the exact type), or where it is strictly necessary (for example, to compute the type of a lambda, etc.).<br />
>
> @evsh <https://github.com/evsh>: Also the efforts at the end should be
> effort, FYI.
>
> (Native English speaker here.)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/qbittorrent/qBittorrent/pull/5083/files/aef92dd0d2be84c71de3afd39fd3e2a3bc610cf7#r59473487>
>